plugins/de/opendiabetes/vault/plugin/importer/libretext/plugin.properties

```diff
@@ -3,3 +3,4 @@ plugin.dependencies=
 plugin.id=LibreTextImporter
 plugin.provider=OpenDiabetes
 plugin.version=0.0.1
+plugin.libs=javacsv.jar
```

Comment:
die sollte doch locker in der base sein wegen dem csvImporter oder nicht?

Reply:
Stimmt